files/image_config/constants/constants.yml

```diff
@@ -32,6 +32,7 @@ constants:
     bbr:
       enabled: true
```

Comment:
Can we set this to 'disabled' in case of storage backend?

Reply:
This file is a common file and not a j2. So setting this to disabled will disable bbr for all roles.

Reply:
agree, i was thinking more like changing this to j2